packages/react/test/specs/commonTests/isConformant.tsx

```diff
@@ -34,6 +34,8 @@ export interface Conformant {
   /** This component uses wrapper slot to wrap the 'meaningful' element. */
   wrapperComponent?: React.ReactType
   handlesAsProp?: boolean
+  /** In the case when a onChange prop name is custom in relation to the controlled prop. */
```

Comment:
Comment is confusing; the way I read the implementation it seems like this key is always required, not just when the prop name is customized.

Reply:
oups, leftover from previous implementation, will correct that.